app/src/main/java/hr/foi/air/servicesync/backend/FirestoreCompanyDetails.kt

```diff
@@ -1,15 +1,21 @@
 package hr.foi.air.servicesync.backend
 
 import android.content.Context
+import android.util.Log
+import com.google.firebase.firestore.FirebaseFirestore
 import com.google.firebase.firestore.GeoPoint
 import hr.foi.air.servicesync.data.CompanyInstance
 
 class FirestoreCompanyDetails {
 
+    private val db = FirebaseFirestore.getInstance()
+
     private var companyName: String = "No name found!"
     private var companyDescription: String = "No description found!"
     private var companyCategory: String = "No category found!"
     private var companyWorkingHours: Int = 0
+    private var companyOpeningTime: String = "0:00"
+    private var companyClosingTime: String = "0:00"
     private var companyGeoPoint: GeoPoint = GeoPoint(0.0, 0.0)
 
     fun loadCompanyName(context: Context, onResult: (String?) -> Unit) {
```

Comment:
This function should be removed completely and refactored to a new, better version